va/proto/va.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto2";
+
+package va;
+option go_package = "proto";
+
+import "core/proto/core.proto";
+
+service VA {
+	rpc IsSafeDomain(IsSafeDomainRequest) returns (IsDomainSafe) {}
+	rpc PerformValidation(PerformValidationRequest) returns (ValidationResult) {}
+}
+
+message IsSafeDomainRequest {
+	optional string domain = 1;
+}
+
+message IsDomainSafe {
+	optional bool isSafe = 1;
```

Comment:
I believe `syntax = 2` will cause *all* fields to be pointers (except slices), not just those that are marked `optional`.